examples/default_level.rs

```diff
@@ -0,0 +1,33 @@
+use clap::Parser;
+use clap_verbosity_flag::{InfoLevel, Verbosity};
+
+/// Foo
+#[derive(Debug, Parser)]
+struct DefaultCli {
+    #[command(flatten)]
+    verbose: Verbosity,
+}
+
+/// Bar
+#[derive(Debug, Parser)]
+struct InfoCli {
+    #[command(flatten)]
+    verbose: Verbosity<InfoLevel>,
+}
+
+fn main() {
+    let error_cli = DefaultCli::parse();
+
+    dbg!(&error_cli.verbose);
+
+    assert_eq!(
+        error_cli.verbose.log_level_filter(),
+        log::LevelFilter::Error
+    );
+
+    let info_cli = InfoCli::parse();
+
+    dbg!(&info_cli.verbose);
+
+    assert_eq!(info_cli.verbose.log_level_filter(), log::LevelFilter::Info);
+}
```

Comment:
imo the asserts distract from the purpose of the example

Reply:
removed